app/incomingCallToast/incomingCallToastPreload.js

```diff
@@ -0,0 +1,30 @@
+const { ipcRenderer } = require('electron');
```

Comment:
exposing the ipcRenderer can bring security risks, see if you can use the www.electronjs.org/docs/latest/api/context-bridge instead. if you get blocked, let me know and I can try to take it from there.